tests/unit/ui/components/TestAnimatedIcon.js

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import AnimatedIcon from 'ui/components/AnimatedIcon';
+import Icon from 'ui/components/Icon';
+
+describe(__filename, () => {
+  it('renders an Icon and sets CSS classes', () => {
+    const root = shallow(<AnimatedIcon name="foo" />);
+
+    expect(root.find(Icon)).toHaveLength(1);
+    expect(root).toHaveClassName('AnimatedIcon');
+    expect(root).toHaveClassName('AnimatedIcon-foo');
+  });
+
+  it('allows a custom className', () => {
+    const root = shallow(<AnimatedIcon name="bar" className="sup" />);
+
+    expect(root).toHaveClassName('AnimatedIcon');
```

Comment:
Do you need these first two assertions? They are already done in the first test. I suppose you may want to make sure that when a custom `className` is passed it doesn't mess with the normal expected `classNames`?

Reply:
Yes, I don't want to replace the default `className`, I want to add it to the list of default ones.